pyccel/ast/cudaext.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+CUDA Extension Module
+Provides CUDA functionality for code generation.
+"""
+from .internals      import PyccelFunction
+
+from .datatypes import (VoidType)
```

Comment:
why the parentheses?

Reply:
No, it's not actually fixed. 26cf144

Reply:
fixed. https://github.com/pyccel/pyccel-cuda/commit/26cf144b7bd775cc1de6cf74ab6287bead55edb1